src/views/assets/Asset/AssetCreateUpdate/BaseAssetCreateUpdate.vue

```diff
@@ -165,13 +174,6 @@ export default {
       const protocolChoices = this.iConfig.fieldsMeta.protocols.el.choices
       protocolChoices.splice(0, protocolChoices.length, ...protocols)
       this.iConfig.fieldsMeta.accounts.el.platform = platform
-      const hiddenCheckFields = ['protocols', 'domain']
-
-      for (const field of hiddenCheckFields) {
-        if (platform[field + '_enabled'] === false) {
-          this.iConfig.fieldsMeta[field].hidden = () => true
-        }
-      }
     }
   }
 }
```

Comment:
This is not relevant to AI programming, please specify whether you need help on coding-related technical issues or improvements in this or similar projects.